src/UriBuilder.Fluent/TerribleDevUriExtensions.cs

```diff
@@ -206,7 +222,10 @@ public static UriBuilder WithHost(this UriBuilder bld, string host)
             {
                 throw new ArgumentNullException(nameof(bld));
             }
-            if(string.IsNullOrWhiteSpace(host)) throw new ArgumentNullException(nameof(host));
+            if (string.IsNullOrWhiteSpace(host))
```

Comment:
I prefer the braces, idk why I didn't do them back then. I think I was just in a hurry to make this thing work pretty well for a project I was working on.

Reply:
great change thanks m8